services/ucs/config_schema.json

```diff
@@ -438,10 +439,9 @@
     }
   },
   "required": [
+    "file_upload_received_topic",
```

Comment:
collection is missing here, but should be required

Reply:
I believe that's because it has a default value. However, in the latest commit ([5589f90](https://github.com/ghga-de/file-services-backend/pull/34/commits/5589f90303626a5c3e6a741a664a322340708bc4)) I removed the default.